css/styles.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+    font-family: Mija;
+    src: url("https://d33wubrfki0l68.cloudfront.net/ac1833dce386eb55d8e105248ef9126622e8e3e9/03f2d/fonts/mijaregular/mija_regular-webfont.woff2") format("woff2"), url("https://d33wubrfki0l68.cloudfront.net/0e0910ceb0e2f9adffd9db31665dafa4ec5c10ff/7214b/fonts/mijaregular/mija_regular-webfont.woff") format("woff");
+}
+
+@font-face {
+    font-family: Mija;
+    src: url("https://d33wubrfki0l68.cloudfront.net/94691bb5497213e09480b17ef68446d2b85667e9/d58ec/fonts/mijabold/mija_bold-webfont.woff2") format("woff2"), url("https://d33wubrfki0l68.cloudfront.net/1d162cc7792bc1f04919a5a344b4082087b9e012/855b4/fonts/mijabold/mija_bold-webfont.woff") format("woff");
+    font-weight: bold;
+}
+
+html,
+body {
+    font-family: Mija, sans-serif;
+    margin: auto;
+}
+
+
+/* GENERAL CONTENT */
+
+.section-header {
+    font-weight: 900;
+    font-size: 1em;
+    letter-spacing: 1px;
+    display: flex;
+    align-items: center;
+}
+
+
+/* .featured-article {
+    display: block;
+    height: 100px;
+    width: 100%;
+    background-color: aqua;
+    border: 1px solid red;
+} */
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

Reply:
I have done this (I should have done, some code cleanup)